elide-contrib/elide-dynamic-config-helpers/src/test/resources/models/tables/table2.hjson

```diff
@@ -20,15 +20,15 @@
         measures : [
             {
             name : highScore
-            type : "{{measure_type}}"
+            type : "INTEGER"
```

Comment:
you can still have it as <%measure_type%>. This way the original test case where this value is replaced from the value in variables.hjson is still valid.